src/pages/MyOffersPage.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Card, makeStyles } from "@material-ui/core";
+import CenteredComponent from "../components/HomePage/CenteredComponent";
+import { MainMask } from "../components/HomePage/MainMask";
+
+const useStyles = makeStyles(() => ({
+    cardContainer: {
+        width: "60%",
+        padding: "24px 72px",
+        boxSizing: "content-box",
+    },
+}));
+
+export const MyOffersPage = () => {
+    const classes = useStyles();
+    return (
+        <>
+            <MainMask />
+            <CenteredComponent>
+                <Card className={classes.cardContainer}>
+                    <h1>My Offers Page</h1>
+                </Card>
+            </CenteredComponent>
+        </>
```

Comment:
By using a PageLayout, you only need to render the content of the Card, ideally in a `<CardContent/>`
Also, PageLayout already handles the card style, so you won't need the styles object probably

```suggestion
<CardContent>
        <h1>My Offers Page</>
</CardContent>
```

Reply:
Also, don't bother having a title for this, since PageLayout will put the title automatically for you!

You can have the content straight away

Reply:
Got it. For now I left with the header since the content will be on another pr